guides/guide-for-adding-windows-node.md

```diff
@@ -25,9 +24,16 @@ Your Kubernetes server must be at or later than version 1.22. To check the versi
 ### Networking Configuration
 
 Once you have a Linux-based Kubernetes control-plane node you are ready to choose a networking solution.
+Below you have 3 possibilities:
+- flannel
+- calico hostprocess
+- flannel hostprocess
+
+>  **Note:** You will need to use only one of them.
 
+## Configuring Flannel on Linux nodes
 
-#### Configuring Flannel with rancher
+>  **Note:** All code snippets in Linux sections are to be run in a Linux environment on the Linux worker node.
 
```

Comment:
Hm, moving forward, I think the hostprocess deployments would be the way to go. Thus, I'd interchange ``flannel`` with ``flannel hostprocess`` sections. I think that could lead to less confusion as well. I was following this guide, going to deploy ``flannel hostprocess``, but ended up following this section as well as I thought this was needed as well, including the Windows bits below, without noticing that the Windows bits below are not for ``flannel hostprocess``.

Or maybe even group up the flannel guide together, since it seems that the Linux side is identical on both cases. The main difference I see is the yamls that have to be applied.

Reply:
Updated it in order to use only flannel hostprocess. I believe this will make it easier to follow and less confusing.